### PR TITLE
Janestreet v0.15 (take 5.5)

### PR DESCRIPTION
**Pull request by aalekseyev**

### CODE REVIEW

File: packages/ocaml-probes/ocaml-probes.v0.15.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+A tool for controlling user-space statically-defined tracing probes for OCaml.
+Experimental.
+"
+url {
+src: "https://ocaml.janestreet.com/ocaml-core/v0.15/files/ocaml-probes-v0.15.0.tar.gz"
+checksum: "sha256=6d154b2f67b6ddd3a501c08b481f5946a02b886554f949a29e7091f6c74f5a90"
+}
+depexts: [
+  depexts: ["linux-headers"] {os-distribution = "alpine"}
+]
```
Comment (kit-ty-kate) [inline packages/ocaml-probes/ocaml-probes.v0.15.0/opam:28]:
```suggestion
depexts: ["linux-headers"] {os-family = "alpine"}
```

File: packages/netsnmp/netsnmp.v0.15.0/opam [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+description: "
+Provides a wrapper around the Net-SNMP client libraries including
+MIB handling and SNMP requests
+"
+url {
+src: "https://ocaml.janestreet.com/ocaml-core/v0.15/files/netsnmp-v0.15.0.tar.gz"
+checksum: "sha256=4c9b6bae3b61d85d92f80fb5c0fe3fd6db498512eeaa3d332eed284f4693d269"
+}
+depexts: [
+  ["net-snmp"] {os-distribution = "arch"}
```
Comment (kit-ty-kate) [inline packages/netsnmp/netsnmp.v0.15.0/opam:31]:
```suggestion
  ["net-snmp"] {os-family = "arch"}
```

File: packages/core_unix/core_unix.v0.15.0/opam [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  "timezone"                 {>= "v0.15" & < "v0.16"}
+  "base-threads"
+  "dune"                     {>= "2.0.0"}
+  "spawn"                    {>= "v0.15"}
+]
+synopsis: "Unix-specific portions of Core"
+description: "
+Unix-specific extensions to some of the modules defined in [core] and [core_kernel].
+"
+depexts: ["linux-headers"] {os-distribution = "alpine"}
```
Comment (kit-ty-kate) [inline packages/core_unix/core_unix.v0.15.0/opam:31]:
```suggestion
depexts: ["linux-headers"] {os-family = "alpine"}
```

File: packages/shell/shell.v0.15.0/opam [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "ppx_jane"   {>= "v0.15" & < "v0.16"}
+  "textutils"  {>= "v0.15" & < "v0.16"}
+  "dune"       {>= "2.0.0"}
+  "spawn"      {>= "v0.12"}
+]
+synopsis: "Yet another implementation of fork&exec and related functionality"
+description: "
+This library is a relatively featureful and flexible implementation of fork&exec with related functionality, including some basic support for asynchronous I/O. This is an old and unmaintained code. New users are advised to try using [Async.Process] for certain use cases and [Shexp_process] for certain other use cases.
+"
+depexts: ["linux-headers"] {os-distribution = "alpine"}
```
Comment (kit-ty-kate) [inline packages/shell/shell.v0.15.0/opam:26]:
```suggestion
depexts: ["linux-headers"] {os-family = "alpine"}
```

**Comment by aalekseyev** [comment]
@kit-ty-kate, is the CI actually still running? It's been two days.

**Comment by kit-ty-kate** [comment]
@aalekseyev yes, it just finished

**Comment by kit-ty-kate** [comment]
Thanks! There are still a few things to fix but they are very minor and can be fixed in subsequent PRs

**Comment by aalekseyev** [comment]
Great, thanks for merging! I was confused by revdeps failing, though (wasn't a previous PR supposed to fix that?)

**Pull request merged**